src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -89,6 +97,9 @@ static GOptionEntry install_option_entries[] = {
   { "touch-if-changed", 0, 0, G_OPTION_ARG_STRING, &opt_touch_if_changed, "Update the modification time on FILE if a new commit was created", "FILE" },
   { "workdir", 0, 0, G_OPTION_ARG_STRING, &opt_workdir, "Working directory", "WORKDIR" },
   { "workdir-tmpfs", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_workdir_tmpfs, "Use tmpfs for working state", NULL },
+#ifdef HAVE_RUST
+  { "yaml-manifest", 0, 0, G_OPTION_ARG_NONE, &opt_yaml_manifest, "Assume the manifest is in YAML format", NULL },
```

Comment:
I think I'd vote we do this automatically if the file ends in `.yaml`, and require that extension just for simplicity.  If we wanted something a bit more robust we could try GLib's MIME sniffing.

Reply:
(I see you did this below...kind of uncertain about the need for an option, but eh.)